src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx

```diff
@@ -83,10 +87,11 @@ export default class SelectFromPhotoLibrary extends React.Component<Props, State
     if (inTesting) {
       return
     }
-
+    console.log(fetchParams)
```

Comment:
Shouldn't the linter have caught this?

Reply:
Nevermind, I see there are others too. Must be temporary.

Reply:
ah, I'll kill those

Reply:
Ah, leaving them in for now 👍